doc/overview.md

```diff
@@ -1,15 +1,18 @@
 Getting started
 ---------------
 
-This application uses npm and Bower to manage its dependencies; npm is used for commandline tools and libraries, and Bower is used to manage browser dependencies.
-When first installing or updating the application, run `npm install` in the terminal to install all dependencies.
+This application uses npm to manage its dependencies; dependencies used in the browser are bundled together using [webpack](https://webpack.github.io).
+When first installing or updating the application, run `npm install` in the terminal to install all dependencies and build a copy of the bundled app in app/appraisal_tab.js.
 
 Running a server
 ----------------
 
-A test server is provided as a part of the project's npm dependencies.
-To run a test server, run `npm start` in the terminal while in the project directory; the demo site will be available at `http://localhost:8000/app/`.
-The port can be customized in the `package.json` file in the root of this repository.
+This application currently can't be run standalone; it can only be run as a part of an Archivematica installation, since it depends on some external JavaScript and some backend APIs.
+To deploy within Archivematica, use the following steps:
+
+1. Run `npm install` to generate a bundled asset.
+2. Copy the `app/index.html` and `app/appraisal_tab.js` files into an Archivematica installation's static file directory.
+   (In future versions of Archivematica, these will be located in the /media/appraisal directory.)
```

Comment:
Does ansible do this currently?

> (In future versions of Archivematica, these will be located in the /media/appraisal directory.)

Unsure what that means - this is where the code will live in the future, or this is where the static files directory will be or something else?